### DISCUSSION TITLE
Build mypage.qmd to mypage/index.html

### DISCUSSION THREAD

**Discussion by th0ger** [quarto-dev/quarto-cli#6870] [Q&A] [answered]
### Description

Hi,

- If I structure my page with subfolders `project/mypage/index.qmd` the served URL will be `mysite.domain/mypage`.
This is a nice end-user URL, but we end-up calling all files `index.qmd` which can be very confusing for the developer, and we also get an additional folder to navigate though.
- If I structure my page as `project/mypage.qmd` the served URL is `mysite.domain/mypage.html`. I would like to avoid the `.html` part.

It is possible the configure Quarto to build `mypage.qmd` into `mypage/index.html`, to get the best of both worlds?

**Comment by cscheid** [comment, marked as answer]
No, unfortunately that's not possible yet. We're planning on some changes in 1.5 that will enable this, but right now that's not possible.

**Comment by mcanouil** [comment]
FYI the related issue is:
- https://github.com/quarto-dev/quarto-cli/issues/2065

The major work mentioned by @cscheid is tracked with:
- https://github.com/quarto-dev/quarto-cli/issues/6518
